symfony/plugins/orangehrmSecurityAuthenticationPlugin/config/doctrine/schema.yml

```diff
@@ -0,0 +1,22 @@
+ResetPasswordLog:
+  tableName: ohrm_secauth_reset_password_log
```

Comment:
what is the standard for ohrm_secauth_ ?

Reply:
This is not a stranded, that should change to ohrm_reset_password_log.I will correct this